index.js

```diff
@@ -0,0 +1,141 @@
+module.exports = {
+  parser: '@typescript-eslint/parser',
+  parserOptions: {
+    ecmaVersion: 2018,
+    ecmaFeatures: {
+      jsx: true,
+    },
+    sourceType: 'module',
+    project: './tsconfig.json',
+  },
+  extends: ['plugin:react/recommended', 'plugin:prettier/recommended'],
+  plugins: ['import', 'unicorn', 'prettier'],
```

Comment:
Extending 'plugin:prettier/recommended' already does everything for you so there's no need to explicitly tell eslint to use the prettier plugin. (https://prettier.io/docs/en/integrating-with-linters.html)